common/log/zap_logger.go

```diff
@@ -54,14 +55,15 @@ type (
 var _ Logger = (*zapLogger)(nil)
 
 // NewTestLogger returns a logger for tests
+// Deprecated: Use testlogger.TestLogger instead.
 func NewTestLogger() *zapLogger {
-	format := os.Getenv(testLogFormatEnvVar)
+	format := os.Getenv(TestLogFormatEnvVar)
 	if format == "" {
 		format = "console"
 	}
 
 	logger := BuildZapLogger(Config{
-		Level:       os.Getenv(testLogLevelEnvVar),
+		Level:       os.Getenv(TestLogLevelEnvVar),
 		Format:      format,
 		Development: true,
```

Comment:
should we port this too? it looks like it controls whether it panics on dpanic

Reply:
Why do we need envs to control test logger? Do we use it anywhere?

Reply:
David does to switch it to JSON format; and I noticed actually my IDE is set up to log on error level instead of debug level.

Reply:
I didn't know we have it.